server/game/core/gameSteps/prompts/ResourcePrompt.ts

```diff
@@ -65,10 +66,10 @@ export class ResourcePrompt extends AllPlayerPrompt {
         Contract.assertNotNullLike(player);
         Contract.assertNotNullLike(card);
 
-        if (!this.activeCondition(player)) {
+        if (!this.activeCondition(player) || !player.hand.includes(card) ||
+          this.selectedCards[player.name].length === this.nCardsToResource) {
```

Comment:
Just for style consistency could you break out the lines like this:

```typescript
  if (
    !this.activeCondition(player) || !player.hand.includes(card) ||
     this.selectedCards[player.name].length === this.nCardsToResource
) {
```

Reply:
ofcourse!